packages/mask/src/plugins/Savings/SNSAdaptor/SavingsDialog.tsx

```diff
@@ -28,11 +28,14 @@ import { SavingsTable } from './SavingsTable'
 import { SavingsForm } from './SavingsForm'
 import type { AaveProtocolDataProvider } from '@masknet/web3-contracts/types/AaveProtocolDataProvider'
 import AaveProtocolDataProviderABI from '@masknet/web3-contracts/abis/AaveProtocolDataProvider.json'
+import { YearnProtocol } from '../protocols/YearnProtocol'
+import type { AbiItem } from 'web3-utils'
+import { flatten, compact, orderBy, sortedUniqBy } from 'lodash-unified'
+import { VaultInterface, Yearn } from '@yfi/sdk'
+import { LDO_PAIRS, YearnChains } from '../constants'
 import { LidoProtocol } from '../protocols/LDOProtocol'
 import { AAVEProtocol } from '../protocols/AAVEProtocol'
-import { LDO_PAIRS } from '../constants'
-import type { AbiItem } from 'web3-utils'
-import { flatten, compact } from 'lodash-unified'
+
 
 function splitToPair(a: FungibleTokenDetailed[] | undefined) {
```

Comment:
The returned type is `any`.

Reply:
do you mean the type of parameter **a** should be **any**?

Reply:
Should avoid using `any`.